### PR TITLE
GH-28: UI Alignment

### PR DESCRIPTION
**Pull request by tamarayustian**
PR addresses these issues:
 - ⁠navbar shouldn't show for login/register
 - user profile information not showing
 - after successful login, redirect to home?
 - rearrange the checkboxes in song search to be a neat grid
 - song list container UI is still a bit messy

### CODE REVIEW

File: ui/src/theme.ts [hunk 1]
```diff
@@ -55,11 +55,11 @@ const customTheme = createTheme({
     },
     h3: {
       fontSize: '1.25rem', // Equivalent to 20px (16 * 1.25)
-      fontWeight: 500,
+      fontWeight: 400,
```
Comment (Midnite1212) [inline ui/src/theme.ts:58]: I think we shod change n align this theme with the actual one we used in the design, not the specs page.

Comment (tamarayustian) [reply ui/src/theme.ts:58]: i was thinking that we can keep it all the same weight in the themes but theyre all different sizes. so we just need to change the weight when we use it.
if wanna do what u suggested, i dont think cxo has the list of all the font size and weights they used, so need to ask them to compile it for us?

Comment (Midnite1212) [reply ui/src/theme.ts:58]: They actly have it in the design specs page. Thats where yy get all the original font weight n sizes. But they just nvr use those lol. Cuz for me if we still hav to adjust the font weight, it kinda defeats the whole purpose of having a theme in the first place :") esp in terms of lines, its the same as using fontweight and font size lol. It just looks slightly better.

Comment (Midnite1212) [reply ui/src/theme.ts:58]: We can ask cxo to compile? Lol

**Review by bhryck — APPROVED** [review]

**Review by Midnite1212 — APPROVED** [review]

**Pull request merged**